lib/talk_helpers.rb

```diff
@@ -6,3 +6,11 @@ def past_and_next_talks items
     i.attributes[:kind] == :talk and i.attributes[:date] <= limit
   end.sort_by { |i| i.attributes[:date] }
 end
+
+def upcoming_talks items
+  limit = Date.today
```

Comment:
Why not? Schedule translates to "Terminplan" in german, and "upcoming" can be translated as "Noch ausstehend" ... so where's the problem?